### ISSUE TITLE
Number of parameters?

### ISSUE THREAD

**Issue by underfitting** [Cadene/pretrained-models.pytorch#59]
Has anyone compared number of parameters?
I tried some good accuracy models, but they are very slow.

In terms of both accuracy and speed, which model is the best one to choose?

**Comment by underfitting** [comment]
Thank you @psavine42 ! It's so useful.
I simply calculated `score = (acc@1)/(log10(n_params))` and top5 models are below:

```
model name | n_params | acc@1
-------------------------------
nasnetamobile | 5289978 | 74.1
densenet121 | 7978856 | 74.6
xception | 22855952 | 78.9
dpn68 | 12611602 | 75.9
se_resnext50_32x4d | 27559896 | 79.1
```
